src/components/input-field/input-field.tsx

```diff
@@ -23,6 +23,7 @@ import {
 } from '../../util/keycodes';
```

Comment:
> The type of the input-field cannot be url because then the full url is required. Therefore, it is set to type = 'text'.

That doesn't seem to be true anymore, so I'm removing those two sentences from the PR description.